packages/dom/src/layout.rs

```diff
@@ -65,9 +65,13 @@ impl LayoutPartialTree for Document {
         compute_cached_layout(self, node_id, inputs, |tree, node_id, inputs| {
             let node = tree.node_from_id_mut(node_id);
 
+            // this is because of hidpi affecting the layout - we need to propagate this down
+            // the 2.0 is the dpi and the .2 is a bit extra since our measure function is a bit off
+            const FUDGE_FACTOR: f32 = 2.2;
+
             let font_metrics = FontMetrics {
-                char_width: 8.0,
-                char_height: 16.0,
+                char_width: 8.0 * FUDGE_FACTOR,
+                char_height: 16.0 * FUDGE_FACTOR,
```

Comment:
Fine since parley is coming soon. The better fudge would be a fixed version of https://github.com/DioxusLabs/blitz/commit/024c26c0ca59945e2469c5c4a863f06b5621141f (which I had to revert because it was breaking the google example. We're currently hardcoding font size for layout but computing for rendering!